### PR TITLE
Update: edit, show condition in template

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#TicketID](https://dev.sun-asterisk.com/issues/xxx)

## What's this PR do ?

- Update condition in template
Screenshot from 2020-08-03 13-23-32
Screenshot from 2020-08-03 13-24-04

### CODE REVIEW

File: public/js/templates.js [hunk 1]
```diff
@@ -39,24 +39,19 @@ function checkData() {
             flag = false;
         }
     });
-    $(".value").each(function () {
```
Comment (hatth-1632) [inline public/js/templates.js:42]: sao xóa vậy em?

File: public/js/templates.js [hunk 2]
```diff
@@ -222,3 +231,87 @@ $(document).ready(function () {
...
+
+        $(field).append(fieldInput);
+        $(operator).append(operatorsDropdown);
+        $(value).append(valueInput);
+        $(btn).append(btnDelete);
+        $(row).append(field, operator, value, btn);
+        $(conditions).append(row);
+    }
+}
+function checkDataCondition() {
```
Comment (hatth-1632) [inline public/js/templates.js:295]: hàm này với hàm checkData() khác gì nhau em? sao k viết chung?

File: resources/views/templates/detail_template_modal.blade.php [hunk 3]
```diff
@@ -9,13 +9,35 @@
...
-                        <label class="field-compulsory fl-left">Name</label>
+                        <label class="field-compulsory p-6 fl-left">Name</label>
                         <input type="text" class="form-control" value="{{ $template->name }}" readonly>
                     </div>
                     <div class="">
                         <label class="field-compulsory p-6 fl-left">Param</label>
                         <textarea class="form-control" rows="10" readonly>{{ $template->params }}</textarea>
                     </div>
+                    <div class="row pdl-15">
+                        <label class="field-compulsory p-6 fl-left">Param</label>
```
Comment (ghost) [inline resources/views/templates/detail_template_modal.blade.php]: Condition chứ

Comment (ghost) [reply resources/views/templates/detail_template_modal.blade.php]: okey :+1:

**Pull request merged**